parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -10,6 +14,27 @@ removeContainerd() {
     apt_get_purge 10 5 300 moby-containerd
 }
 
+removeCurl() {
+    apt_get_purge 10 5 300 curl
+}
+
+installLatestCurlManually() {
+    # Too Risky to manage within the components.json file since curl is required for the bootstrap logic to work
+    # currently this logic is only needed for 18.04, we will be able to remove this logic once we no longer support 18.04
+    
+    # renovate: datasource=github-releases depName=curl/curl
+    version="8.9.0"
+    deb_file="/tmp/curl.deb"
+    removeCurl || exit $ERR_CURL_REMOVE_TIMEOUT
```

Comment:
this will be called with `installDeps`, which is called at build-time + provisioning time

we should probably add some logic here to check if a sufficient version curl is already installed, otherwise we'd be increasing provisioning latency across the board

Reply:
fixed!